src/util/cookie.ts

```diff
@@ -4,6 +4,8 @@ export interface Cookie {
     value: string;
 }
 
+export type SameSite = 'Strict' | 'Lax' | 'None';
```

Comment:
It might to be a good idea to define `SameSite` as a String enum instead of union